src/test-interp.cc

```diff
@@ -140,7 +139,7 @@ TEST_F(InterpTest, Disassemble) {
   auto buf = stream.ReleaseOutputBuffer();
 
   ExpectBufferStrEq(*buf,
-R"(   0| alloca 1
+                    R"(   0| alloca 1
```

Comment:
This seems worse.   Maybe we can can `ExpectBufferStrEq(*buf, R"(\` to avoid the issue?

Reply:
Done

Reply:
Done. Also fixed places with similar patterns in this file. By the way what is `\` at the end for?

Reply:
Hmm, I guess `\` doesn't work in raw strings...

Reply:
Hmm, I don't think there's an easy way around this. `\` doesn't work as multiline-splacer because in raw strings `\` is considered as just a literal `\`. I agree it's ugly, but [the existing code](https://github.com/WebAssembly/wabt/blob/6c9355e490a37c7a990d504f32676c9d7a2ebb13/src/test-interp.cc#L142-L161) does not completely line up either. Also we have multiple raw strings like this in this file so adding `// clang-format off` to all those places seems a little too much. How about just living with this change, even if it is a little ugly?

Reply:
Can we just revert these parts of the PR?  Or are you committed to having 100% of the codebase be clang-format clean?    Maybe we could punt on these chunks for now?

Reply:
Reverted this and two more parts that use a raw string and have a similar change.